modules/gcp_bigquery_dataset_iam_policy/README.md

```diff
@@ -0,0 +1,37 @@
+<!-- BEGIN_TF_DOCS -->
```

Comment:
If this is a separate module we should at least have a few lines describing what it does